build/tools/SharedCodeGenerator/Outputs/ShellOutput.cs

```diff
@@ -53,7 +53,7 @@ public string GetContent()
             if (this.collection.ListConstants?.Any() ?? false)
             {
                 // exclude runtime versions constants lists, as they will just overwrite each other in shell
-                foreach (var constant in this.collection.ListConstants.Where(c => !c.Key.Equals("runtime-versions")))
+                foreach (var constant in this.collection.ListConstants.Where(c => !c.Key.Equals("runtime-versions") && !c.Key.Equals("fpm-runtime-versions")))
```

Comment:
We cannot use `.Contains` or `.EndsWith`, in the loop, since it will affect changes in this file:
![image](https://github.com/microsoft/Oryx/assets/86327553/6ad918f2-587f-40fa-8422-d37cf9fc3e70)

Reply:
Ah, that makes sense -- if we decide to skip anything in the future, it may make sense to abstract the "strings to ignore" into a separate variable and iterate over those with LINQ, but this is good for now.

Reply:
@william-msft can you please add a little bit more hint on how it will gonna affect the changes in the mentioned file?
other than that looks good.